src/pages/our-leadership.mdx

```diff
@@ -0,0 +1,72 @@
+# oSTEM Executive Committee and Board of Directors
+
+
+![Image of Lilian Martinez](images/lilian-martinez.jpg "Lilian Martinez")
+## Lilian Martinez
+**Executive Director & CEO**
+
+Email: lilian.martinez@ostem.org
+
+---
+![Image of Matthew Welmers](images/matthew-welmers.jpg "Matthew Welmers")
+## Matthew Welmers
+**Annual Conference Chair**
+
+Email: matthew.welmers@ostem.org
+
+---
+![Image of William Agnew](images/william-agnew.jpg "William Agnew")
+## William Agnew
+**Vice President External STEM Partnerships**
+
+Pronouns: he/him
+
+Email: william.agnew@ostem.org
+
+--- 
+![Image of Sindhu Sreedhar](images/sindhu-sreedhar.jpg "Sindhu Sreedhar")
+## Sindhu Sreedhar
+**Vice President Membership**
+
+Pronouns: she/her
+
+Email: Sindhu.Sreedhar@ostem.org
+
+Sindhu Sreedhar is an engineer at Toyota R&D in Michigan, working on process engineering for seats 
+and seat parts. She serves as the vice-chair of Toyota's LGBTQ+ employee resource group and 
+coordinates the sponsorship of Ann Arbor Pride. Sindhu holds a BSE in Chemical Engineering from the 
+University of Michigan, where she lead the University of Michigan oSTEM chapter. In her free time, 
+Sindhu is training to become a yoga teacher and hopes to specialize in yoga for QTPOC and 
+decolonizing yoga.
+
+---
+![Image of Philip Thakadiyil](images/philip-thakadiyil.jpg "Philip Thakadiyil") 
+## Philip Thakadiyil
+**Vice President Finance**
+
+Pronouns: he/him
+
+Email: philip.thakadiyil@ostem.org
+
+---
+![Image of Travis DePrato](images/travis-deprato.jpg "Travis DePrato")
+## Travis DePrato
+**Vice President Innovation & Technology**
+
+Email: travis.deprato@ostem.org
+
+---
+![Image of Amy Phan](images/amy-phan.jpg "Amy Phan")
+## Amy Phan
```

Comment:
lol

Reply:
Can you reach out to @janine9vn to get a headshot? You can leave the bio empty for now.

Reply:
(for context: Janine is the current VP of MarComms, Amy Phan is the previous, the old site was just never updated)